MaxText/configs/base.yml

```diff
@@ -19,7 +19,7 @@ run_name: ""
 # Types of models supported.
 # default model (palm like model with no overrides in config)
 model_name: "default"
-rms_norm_epsilon: 1.e-06
+norm_epsilon: 1.e-06
```

Comment:
Agreed rms_norm_epsilon is the wrong name without rms_norm's but norm_epsilon isn't right. How about "normalization_layer_epsilon"?

Reply:
Agreed.

Reply:
Done.